src/components/vr_cooperation/include/vr_cooperation/vr_module.h

```diff
@@ -200,9 +206,14 @@ class VRModule : public functional_modules::GenericModule,
   bool IsVRServiceSupported() const;
 
   /**
-   * @brief send response message to mobile in case VRService is not supported
+   * @brief send response message to mobile
+   * @param msg
+   * @param success true if successful; false, if failed
+   * @param result Mobile result
+   * @param info Provides additional human readable info regarding the result(may be empty)
    */
-  void SendUnsupportedServiceResponse(application_manager::MessagePtr& msg);
+  void SendResponseToMobile(application_manager::MessagePtr msg, bool success,
```

Comment:
Method which will send response for register/unregister request, could received only message